xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/transactions/UriTest.java

```diff
@@ -0,0 +1,29 @@
+package org.xrpl.xrpl4j.model.transactions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class UriTest {
+
+  @Test
+  public void uriEquality() {
```

Comment:
Can you add a test here using a non ipfs URI? Probably just an HTTP/S URL for posterity's sake